aiorospy/CMakeLists.txt

```diff
@@ -9,14 +9,15 @@ catkin_package()
 
 catkin_python_setup()
 
-install(FILES requirements.txt
+install(FILES requirements.in
```

Comment:
Is `.in` a standard or convention from somewhere?

Reply:
it is in pip-tools, but i now regret that decision.

I'm going to switch to `requirements.txt` and `Pipfile.lock`, which is standard-ish and supported by github's vulnerability scanner

Reply:
it's the default from `pip-tools` (https://github.com/locusrobotics/catkin_virtualenv/pull/55/files#diff-04c6e90faac2675aa89e2176d2eec7d8R117).

One advantage of keeping the locked dependencies in `requirements.txt` is that it'll get scanned by tools such as github's vulnerability scanner.